src/core_plugins/input_control_vis/public/components/vis/list_control.js

```diff
@@ -26,25 +26,43 @@ export class ListControl extends Component {
     return `${selected.label.substring(0, 23)}...`;
   }
 
+  renderControl() {
+    if (!this.props.control.isEnabled()) {
+      // react-select clobbers the tooltip, so just returning a disabled input instead
+      return (
+        <input
+          disabled={!this.props.control.isEnabled()}
```

Comment:
nit: this is already inside the `if (!this.props.control.isEnabled()) {` i'd probably just do `disabled={true}`